chaosaws/cloudwatch/probes.py

```diff
@@ -1,11 +1,10 @@
 # -*- coding: utf-8 -*-
 from datetime import datetime, timedelta
 from statistics import mean
-
+from typing import Any, Dict, List
```

Comment:
an extra line between standat libs imports and 3rd party ones is conventionnal

Reply:
Hi @Lawouach, should I just add an empty line? Or should I change something else?

Reply:
just an empty line after the typing import would be superb :)

Reply:
ok, corrected!